tbx/work/models.py

```diff
@@ -266,83 +265,95 @@ def read_time(self):
 
 
 # Work index page
-@method_decorator(get_default_cache_control_decorator(), name="serve")
 class WorkIndexPage(SocialFields, Page):
-    template = "patterns/pages/work/work_listing.html"
+    template = "patterns/pages/work/work_index_page.html"
```

Comment:
Just out of curiosity, what was the reason for the name change here?

Reply:
- In order to maintain consistency by using the class name as the basis for the template name (snake_case version of the class name). Ref: https://github.com/torchbox/torchbox.com/pull/51#discussion_r1466164142
- Since we already have `tbx/project_styleguide/templates/patterns/organisms/work-listing/work-listing.html`, I thought it may be confusing to have two similar templates.

Reply:
ah ok - but I think that we are removing that template, and also 'listing' is a bit more consistent with other names. But not a big deal now.